src/main/java/Luke.java

```diff
@@ -0,0 +1,205 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Luke {
+    public static void main(String[] args) throws LukeException {
+        // Greetings
+        System.out.println("________________________________________________________________________");
+        System.out.println("Hello! I'm Luke");
+        System.out.println("What can I do for you?");
+        System.out.println("________________________________________________________________________");
+
+        // User inputs
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+        ArrayList<Task> list = new ArrayList<>();
+
+        // Conditions
+        while (!input.equals("bye")) {
+            if (input.equals("list")) {
```

Comment:
Maybe consider creating functions to execute the commands ? E.g. when input.equals("list ") it call a separate function like outputTasks(). You can possibly perform you validation checks in these functions as well.